tests/functional/registry/test_deployment.py

```diff
@@ -2,14 +2,21 @@
 
 
 def test_deployment_management(
-    gov, guardian, rewards, registry, Vault, create_token, create_vault
+    gov, guardian, rewards, registry, Vault, create_token, create_vault, rando
 ):
     token = create_token()
 
     # No deployments yet for token
     with brownie.reverts():
         registry.latestVault(token)
 
+    # Check that newRelease raises if vault governance is a rando
+    bad_vault = create_vault()
+    bad_vault.setGovernance(rando)
+    bad_vault.acceptGovernance({"from": rando})
+    with brownie.reverts():
+        registry.newRelease(bad_vault, {"from": gov})
+
```

Comment:
Minor nit, place this in `test_release_management` in the other file since this has to do with releases

Reply:
Makes sense. Moved.